### PR TITLE
vim, wget, tldr, yarn, which, where, and 10 more: add Chinese translation

### PR DESCRIPTION
**Pull request by KsRyY**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by KsRyY** [comment]
More translations:
- zsh
- zstd
- view
- vue
- virtualenv

**Comment by KsRyY** [comment]
More translations:
- yarn-why
- z
- w

**Comment by KsRyY** [comment]
> I don't know Chinese but merge when ready.

Thanks.

**Comment by KsRyY** [comment]
Google Translation may be helpful

**Comment by KsRyY** [comment]
More transtaion:
- write
- curl (partial)

**Comment by KsRyY** [comment]
More translation: 
- atom

File: pages.zh/common/curl.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+- 发送带有额外请求头, 使用自定义请求方法的请求:
+
+`curl -H {{'X-My-Header: 123'}} -X {{PUT}} {{http://example.com}}`
+
+- 发送 JSON 格式的数据, 并附加正确的 `Content-Type` 请求头:
+
+`curl -d {{'{"name":"bob"}'}} -H {{'Content-Type: application/json'}} {{http://example.com/users/1234}}`
+
+- 为服务器授权传入用户名和密码:
```
Comment (einverne) [inline pages.zh/common/curl.md]: How about "使用用户名和密码，授权访问服务器"?

File: pages.zh/common/wget.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+- Download a file from an HTTP server using Basic Auth (also works for FTP):
+
+`wget --user={{username}} --password={{password}} {{https://example.com}}`
+
+- 继续一个未完成的下载任务:
+
+`wget -c {{https://example.com}}`
+
+- 将指定文件中所有列出的URL下载到一个地址中:
```
Comment (einverne) [inline pages.zh/common/wget.md]: 将指定文件中所有列出的URL下载到一个**目录**中

**Comment by KsRyY** [comment]
@einverne Thanks for your suggestion!

**Comment by KsRyY** [comment]
More translation:
- wget(finished)
- curl(finished)

**Comment by sbrl** [comment]
Looks like @einverne approves and we've got 2  other approvals too, so I'll merge this.

**Pull request merged**